tests/src/tools/ReadyToRun.SuperIlc/Application.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+namespace ReadyToRun.SuperIlc
+{
+    public class Application
+    {
+        private List<string> _compilationInputFiles;
+
+        private string  _mainExecutable;
+
+        private readonly List<ProcessInfo[]> _compilations;
+
+        private readonly ProcessInfo[] _execution;
+
+        public Application(
+            List<string> compilationInputFiles, 
+            string mainExecutable, 
+            IEnumerable<CompilerRunner> compilerRunners, 
+            string coreRunPath)
+        {
+            _compilationInputFiles = compilationInputFiles;
+            _mainExecutable = mainExecutable;
+
+            _compilations = new List<ProcessInfo[]>();
+
+            ;            foreach (string file in _compilationInputFiles)
```

Comment:
Something weird happened here

Reply:
Ahh, sorry about the formatting error, will fix; thanks for spotting.